qa/tasks/cephfs/filesystem.py

```diff
@@ -272,11 +272,15 @@ def get_nonnumeric_values(value):
                      "-Infinity": -float("inf")}
                 return c[value]
 
-            j = json.loads(response_data.replace('inf', 'Infinity'),
-                           parse_constant=get_nonnumeric_values)
-            pretty = json.dumps(j, sort_keys=True, indent=2)
-            log.debug(f"_json_asok output\n{pretty}")
-            return j
+            
+            try:
+                j = json.loads(response_data.replace('inf', 'Infinity'),
+                            parse_constant=get_nonnumeric_values)
```

Comment:
how about we just focus exception handling to only this line since this is where the exception will occur.
```
j = ""
try:
    j = json.loads(response_data.replace('inf', 'Infinity'),
                parse_constant=get_nonnumeric_values)
except json.decoder.JSONDecodeError:
    raise RuntimeError(response_data)

pretty = json.dumps(j, sort_keys=True, indent=2)
log.debug(f"_json_asok output\n{pretty}")
return j
```

since it is anyways not going to run through `json.dumps() `, this way we can make sure we don't suppress error if by any chance we receive a json response with invalid schema in `pretty = json.dumps(j, sort_keys=True, indent=2)`